linchpin/api/fetch/__init__.py

```diff
@@ -0,0 +1,10 @@
+from fetch_local import FetchLocal
+from fetch_http import FetchHttp
+from fetch_git import FetchGit
+
+FETCH_CLASS = {
+        "FetchLocal" : FetchLocal,
+        "FetchHttp": FetchHttp,
+        "FetchGit": FetchGit
+        }
+__all__ = ["FetchLocal", "FetchHttp", "FetchGit"]
```

Comment:
What does this do? I don't see any code checking it to make sure of anything useful. Does it need to be here?